client/scripts/rails_only.jsx

```diff
@@ -8,3 +8,7 @@ require('es5-shim/es5-sham');
 // Due to issue: https://github.com/ariya/phantomjs/issues/12401
 // Phantomjs does not like promises
 require('es6-promise').polyfill();
+
+require('expose?jQuery!jquery');
+require('expose?$!jquery');
+require('expose?jquery-ujs!jquery-ujs');
```

Comment:
@alexfedoseev Thanks for the example! 

If I comment out ```require('expose?jquery-ujs!jquery-ujs')``` that I get: ```ActionController::InvalidAuthenticityToken (ActionController::InvalidAuthenticityToken)``` webserver error.

With it left as is, there are not CSRF issues and everything works properly.

So @justin808 @alexfedoseev why is inclusion in an ```entry``` preferred over the way is it exposed now? Thanks, for the education!